common/api/ecschema-rpcinterface-impl.api.md

```diff
@@ -0,0 +1,21 @@
+## API Report File for "@bentley/ecschema-rpcinterface-impl"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { ECSchemaRpcInterface } from '@bentley/ecschema-rpcinterface-common';
+import { IModelRpcProps } from '@bentley/imodeljs-common';
+import { SchemaKey } from '@bentley/ecschema-metadata';
+
+// @internal
+export class ECSchemaRpcImpl extends ECSchemaRpcInterface {
+    getSchemaJSON(tokenProps: IModelRpcProps, schemaName?: string): Promise<string>;
```

Comment:
You may need to re-run extract-api to get the updated param.

Reply:
yup! its currently running, just taking a while.